requirements.txt

```diff
@@ -4,4 +4,5 @@ pennylane>=0.34
 pybind11
 pytest~=8.0.0
 pytest-cov
-pytest-mock
\ No newline at end of file
+pytest-mock
+pytest-xdist
```

Comment:
We typically use `requirements-dev.txt` to install dev/ci deps, so I don't think this is required. Doesn't hurt either. I think `pytest` is only included here so that PennyLane's `pl-device-test` can run.

Reply:
yeah, I usually run `requirements.txt` in a fresh container before checking lightning unit tests and it may be good to also install `pytest-xdist` (along with `pytest`) to boost up `pl-device-test`ing even for developers.